.gitignore

```diff
@@ -211,4 +211,5 @@ FakesAssemblies/
 GeneratedArtifacts/
 _Pvt_Extensions/
 ModelManifest.xml
-/Master/NucleusCoop.VC.VC.opendb
+**/*.VC.VC.opendb
+**/*.VC.db
```

Comment:
I did, but while I was adding "\*.VC.db" to the .gitignore file, I thought I should generalize both patterns, so that they would include any directory.